### DISCUSSION TITLE
Frequency Manager

### DISCUSSION THREAD

**Discussion by bubbaearle** [AlexandreRouma/SDRPlusPlus#763] [Q&A]
Where are the file(s) / list(s) stored that are created when you save a frequency?

Are they single or multiple files?

Can they be edited with Excel or something similar?

I'd like to be able to quickly create & share such lists between multiple machines if possible.

**Comment by OttoPattemore** [comment]
The frequencies are stored in the frequency manager's config file.  It is located in SDR++'s root directory in the file ``frequency_manager_config.json``.  A search for this in your file manager should find it on your system.  Using a simple python script you could convert it to a CSV file for Excel.

There is also an export function built into SDR++. Select the bookmarks you want ( use shift to select multiple ) and press export. This will be exported as json.  Another machine can then import the json file.

Sample from the frequency_manager_config.json file:
```json
{
    "bookmarkDisplayMode": 1,
    "lists": {
        "General": {
            "bookmarks": {
                "Example bookmark": {
                    "bandwidth": 15000.0,
                    "frequency": 11090000.0,
                    "mode": 2
                }
            },
            "showOnWaterfall": true
        }
    },
    "selectedList": "General"
}

```
